ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -4,6 +4,9 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz --output oc.tar.gz
```

Comment:
Shouldn't this already be apart of the image we are using?

Reply:
Wasn't sure it was. I'll remove it thanks!